tools/Makefile

```diff
@@ -41,3 +41,9 @@ new-release:
 update-version:
 	## When manually updating major version
 	./releaser/scripts/update-version.sh
+
+.PHONY: generate_mocks
+generate_mocks:	
+	go install github.com/vektra/mockery/v2@v2.42.1
```

Comment:
Antipattern. If we need mockery installed let's keep it at dependency level so we can get it automatically updated etc.

Reply:
Let's make follow up PR.

Reply:
ok